proto/dymension/eibc/demand_order.proto

```diff
@@ -0,0 +1,28 @@
+syntax = "proto3";
+package dymensionxyz.dymension.eibc;
+
+import "gogoproto/gogo.proto";
+import "dymension/common/common.proto";
+import "cosmos/base/v1beta1/coin.proto";
+
+option go_package = "github.com/dymensionxyz/dymension/v3/x/eibc/types";
+
+message DemandOrder {
+    // id is a hash of the form generated by GetRollappPacketKey, 
+    // e.g status/rollappid/packetProofHeight/packetDestinationChannel-PacketSequence which gurantees uniqueness
```

Comment:
doesn't the rollappID and packetDestinationChannel are the same?

Reply:
no. rollapp Id is usually the id of the rollap (e.g omri-12312) an dest channel is of the form : `channel-12`. unless you mean something else by same.